packages/shorebird_cli/lib/src/cache.dart

```diff
@@ -205,6 +231,9 @@ class AotToolsDillArtifact extends CachedArtifact {
   @override
   String get storageUrl =>
       '${cache.storageBaseUrl}/${cache.storageBucket}/shorebird/${shorebirdEnv.shorebirdEngineRevision}/$name';
+
+  @override
+  String? get checksum => null;
```

Comment:
Should we add the checksum to the other artifacts in follow up?

Reply:
I'm not sure?  For artifacts that are tied to a Flutter version (rather than a Shorebird version) we would have to have them in the manifest.yaml on storage rather than checked into the code.

Reply:
But then, don't we run the risk of the manifest file getting corrupted as well?